explainaboard/loaders/file_loader.py

```diff
@@ -145,7 +145,7 @@ def from_dict(cls, data: dict) -> FileLoaderMetadata:
                 for k1, v1 in data['custom_features'].items()
             }
         if 'custom_analyses' in data:
-            custom_analyses = [Analysis.from_dict(v) for v in data['custom_analyses']]
+            custom_analyses = data['custom_analyses']
```

Comment:
I'm confused about why this change was made. Shouldn't we deserialize the dictionaries when we read them in from the file?

Reply:
It will bring some errors due to twice-deserialization: https://github.com/neulab/ExplainaBoard/blob/fa21e64e54341cd1554d27ba78fa139eeed1fc30/explainaboard/processors/processor.py#L225

Reply:
OK, sounds good. Hopefully this will be fixed by the new PRs on making deserialization/serialization better.